ui/components/component-library/banner-tip/README.mdx

```diff
@@ -0,0 +1,147 @@
+import { Story, Canvas, ArgsTable } from '@storybook/addon-docs';
+import { BannerTip } from './banner-tip';
+import { BannerBase } from '..';
+
+# BannerTip
+
+`BannerTip` is an inline notification that notifies users of important information & sometimes time-sensitive changes.
+
+<Canvas>
+  <Story id="components-componentlibrary-bannertip--default-story" />
+</Canvas>
+
+## Props
+
+The `BannerTip` accepts all props below as well as all [Box](/docs/components-ui-box--default-story#props) component props
+
+<ArgsTable of={BannerTip} />
+
+The `BannerTip` accepts all `BannerBase` component props below
+
+<ArgsTable of={BannerBase} />
+
+### Logo Type
+
+Use the `logoType` prop with the `BANNER_TIP_LOGOS` object from `../../component-library` to change the context of `BannerTip`.
+
+Possible options:
+
+- `BANNER_TIP_LOGOS.GREETING` Default
+- `BANNER_TIP_LOGOS.CHAT`
```

Comment:
I preferred Variant because that was the prop it was describing, if you are describing a Type then Type is appropriate, but if you are describing something and expressing it as a TypeScript type, appending Type to it is redundant and is usually being done to avoid namespace conflicts. 

Summary: BannerTipLogoType is great.